test/kino/download_test.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Kino.DownloadTest do
+  use ExUnit.Case, async: true
+
+  describe "start/3" do
+    test "returns %Kino.JS{} " do
```

Comment:
Annotation: I didn't really know what to test here, as-is this test file might not be very useful.

Reply:
Yeah, there's not much to test with plain `Kino.JS` components, since the more relevant part is on the JS side. Once we have `Kino.JS.Live`, we can test the messages :)